fix: support for flow types

whoshallsucceed opened the pull request:
Before this patch, ctags would not recognize simple import type
statements such as:

```
import type impFlowType from "./ImportDefault";
import type { impFlowType } from "./ImportDefault";
```

AdrienGiboire commented:
I just noticed that somehow I did not have the most recent version of this :thinking:

Commit:
fix: support for flow types

Before this patch, ctags would not recognize simple import type
statements such as:

```
import type impFlowType from "./ImportDefault";
import type { impFlowType } from "./ImportDefault";
```

romainl commented:
Thank you for this, but direct imports/exports are not tagged, either for straight ES6 imports/exports or for Flow. Only named imports/exports are.

This is on purpose:

* `Foo` is defined in `Foo.js`, exported without renaming, and imported without renaming. If we tagged direct imports and direct exports we would have three tags for the same symbol, only one of which being interesting. Since we only tag named imports and named exports, we only have one tag, for the definition.

* `Foo` is defined in `Foo.js`, exported as `Bar`, and imported without renaming. If we tagged direct imports we would have one tag for `Foo` and two tags for `Bar`. Since we only tag named imports, we have two tags and we can follow the trail.

* `Foo` is defined in `Foo.js`, exported as `Bar`, and imported as `Baz`. It is a silly practice but well… since we tag named imports and named exports it is still possible to follow the trail.
